admin/app/services/oidc-identity-providers.js

```diff
@@ -11,4 +11,11 @@ export default class OidcIdentityProviders extends Service {
     const oidcIdentityProviders = await this.store.findAll('oidc-identity-provider');
     oidcIdentityProviders.map((oidcIdentityProvider) => (this[oidcIdentityProvider.id] = oidcIdentityProvider));
   }
+
+  async loadAllReadyIdentityProviders() {
```

Comment:
La remarque de @clemlatz m'a fait me pencher sur ce nommage. Le nommage `loadAllReadyIdentityProviders` est un peu problématique car il diffère de la logique de nommage choisie pour le back. Dans le back on a : 
* `getAllOidcProviderServices`/`getAllIdentityProvidersForAdmin`
* `getReadyOidcProviderServices`

On voit bien la dichotomie _all_ vs _ready_. Si on a un nommage _allReady_, tout est mélangé.

Quant à _onlyReady_ ça sur-spécifie peut-être un peu mais surtout ça ne suit pas la dichotomie présente dans le back.
```suggestion
  async loadReadyIdentityProviders() {
```